packages/modules/vehicles/mercedeseq/auth.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
+
+import json
+import sys
+import time
+import html
+import paho.mqtt.subscribe as subscribe
+import paho.mqtt.publish as publish
+sys.path.append("../../../")
+from modules.common import req  # noqa: E402
+
+# call parameters
+ev_id = str(sys.argv[1])
+code = str(sys.argv[2])
+
+moddir = '/var/www/html/openWB/packages/modules/vehicles/mercedeseq/'
+
+
+def printDebug(message, level):
+    htmlmsg = html.escape(message)
+    if level >= debug:
+        print("<p>" + htmlmsg + "</p>")
+
+
+def printHtml(message):
+    htmlmsg = html.escape(message)
+    print("<p>" + htmlmsg + "</p>")
+
+
+print("<html>")
+
+msg = (subscribe.simple("openWB/system/debug_level", hostname="localhost"))
+debug = int(str(msg.payload.decode("UTF-8")))
```

Comment:
`decode()` gibt bereits einen String zurück.
```suggestion
debug = int(msg.payload.decode("UTF-8"))
```